docs/reference/modules/admin-bar.md

```diff
@@ -0,0 +1,126 @@
+
+### `groups`
+
+The `groups` option takes an array of one or more objects that group several menu items together in the admin bar as a dropdown menu. Each of the `groups` objects requires a `label` and an array of menu `items`. The `label` will be used as the label displayed in the menu. The `items` array contains the names of the individual menu items you want to appear in the dropdown, entered in the order you want them to appear. Note: Menu names for `piece-type` items are the name of the piece-type, not the label. For core items, like 'Images', the name is prefixed - '@apostrophecms/image'.
+
+**Example**
+
+<AposCodeBlock>
+
+```javascript
+
+module.exports = {
+ options: {
+   groups: [
+     {
+       name: 'media',
+       label: 'Media',
+       items: [
+         '@apostrophecms/image',
+         '@apostrophecms/file',
+         '@apostrophecms/image-tag',
+         '@apostrophecms/file-tag'
+       ]
+     }
+   ]
+ }
+};
+```
+<template v-slot:caption>
+    modules/@apostrophecms/admin-bar/index.js
+  </template>
+</AposCodeBlock>
+
+This will result in grouping those four core modules into a single dropdown menu displayed as 'Media' on the menu bar.
+![ApostropheCMS admin bar with open dropdown menu titled 'Media'](/images/group-menu.png)
+
+## Featured methods
+
+The following method belongs to this module and may be useful in project-level code. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/admin-bar/index.js) for all methods that belong to this module.
+<!-- Some are used within the module and would just create noise here. -->
+
+Because this module has an alias, you can call these from another module from the alias path. For example, `self.apos.adminBar.add()`.
+
+### `add(name, label, permission, options)`
+
+Add an item to the menu bar.
+
+The `name` for the item must be unique within the menu bar to avoid conflicts. When the menu item is clicked, the `name` argument will be emitted on `apos.bus` as the value of an `admin-menu-click` event. If this item controls a specific modal, this will be caught by `TheAposModals` to display the correct modal. If this is the case, `name` should be the module name with a `:editor` or `:manager` suffix. For example, `@apostrophecms/global:editor`. A more complex example is contained within the [`@apostrophecms/login` module](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/login/ui/apos/apps/AposLogin.js).
+
+**Example**
+
+Code added to handle click events on the new button in the admin UI frontend:
+
+<AposCodeBlock>
+
+```javascript
+export default () => {
+  apos.bus.$on('admin-menu-click', async (item) => {
+    // Make sure it is the button we care about, leave others to their own handlers
+    if (item !== 'myCustomModule') {
+      return;
+    }
+    // Custom code for button action
+  };
+}
+```
+<template v-slot:caption>
+/modules/myCustomModule/ui/apos/index.js
+</template>
+</AposCodeBlock>
+
```

Comment:
Missing closing backticks

Reply:
I'm not sure I get this one. There are backticks before the opening `<template>` tag. Linting was slightly off here and in the previous example.

Reply:
Seems fixed now.